services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events.py

```diff
@@ -103,7 +104,9 @@ async def will_trigger(cls, app: FastAPI, scheduler_data: SchedulerData) -> bool
         )
 
     @classmethod
-    async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
+    async def action(  # pylint:disable=too-many-statements
```

Comment:
probably this linter error that you are disabling is a warning that your code is getting too complex!

Reply:
OK, took the opportunity to refactor a bit